beta/examples/tensorflow/common/object_detection/evaluation/coco_evaluator.py

```diff
@@ -227,10 +227,6 @@ def update(self, predictions, groundtruths=None):
                 groundtruths['height'] = sizes[:, 0]
                 groundtruths['width'] =  sizes[:, 1]
 
-            if 'num_detections' not in groundtruths:
```

Comment:
Could you comment on why we need these changes?

Reply:
This code is not necessary now. In this place, `'num_detection'` always is in `groundtruths` and already has the required format.